build.gradle

```diff
@@ -248,7 +248,8 @@ project(':ambry-network') {
         compile project(':ambry-api'),
                 project(':ambry-utils'),
                 project(':ambry-commons'),
-                project(':ambry-clustermap')
+                project(':ambry-clustermap'),
+                project(':ambry-protocol')
```

Comment:
It's used for a metric:

```
      RequestOrResponse request = (RequestOrResponse) (requestInfo.getRequest());
      long waitingTime = streamInitiateTime - request.requestCreateTime;
      http2ClientMetrics.requestToNetworkClientLatencyMs.update(waitingTime);
```

I can remove this metric if it's not clean to import `ambry-protocol`.

Reply:
It is probably okay to import since that dependency is probably needed by anyone that uses the network client anyways. One other alternative would be to add a `getRequestCreationTime()` method to the `SendWithCorrelationId` interface. I don't have a strong preference for either direction though.